src/deploy.rs

```diff
@@ -267,7 +280,9 @@ pub async fn deploy_profile(
             a => return Err(DeployProfileError::SSHActivateExitError(a)),
         };
 
-        info!("Success activating, done!");
+        if !dry_activate {
+            info!("Success activating, done!");
```

Comment:
Might be worth adding a dry-activate equivilent of this (and the "Activating profile" message above) instead of just removing